src/IO/ZarrMultiscaleSpatialImage.js

```diff
@@ -87,13 +87,11 @@ const makeCoords = ({ shape, multiscaleImage, dataset }) => {
 const createScaledImageInfo = ({
   multiscaleImage,
   pixelArrayMetadata,
-  pixelArrayAttrs,
   dataset,
 }) => {
   const dims =
-    pixelArrayAttrs?._ARRAY_DIMENSIONS ?? // xarray
```

Comment:
Should we just change the order here -- first try / trust `axes`, then `_ARRAY_DIMENSIONS`?

Reply:
Seems that many NGFFs don't have a foo.zarr/s0/.zattrs where `_ARRAY_DIMENSIONS` lives.  To avoid 404s the code is not fetching the scale level `.zattrs`, so no `_ARRAY_DIMENSIONS` to fall back to.